Assets/Tiled2Unity/Prefabs/Core - Pont.prefab

```diff
@@ -103,7 +103,7 @@ Transform:
   m_LocalPosition: {x: 0, y: 0, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_Children:
-  - {fileID: 4766272462590212}
```

Comment:
Why were these prefab files changed at all?

If you didn't mean to modify them, might as well restore them as they were before!

Reply:
Thank you for noticing.
These changes were indeed unintentional and I reverted the files back.